RELEASE_NOTES.md

```diff
@@ -1,3 +1,45 @@
+## Cookstyle 6.15
+
+### RuboCop 0.89.1
+
+RuboCop has been updated to 0.89.1, which offers significant performance and autocorrection improvements to many Ruby cops that Cookstyle utilizes.
+
+### New Ruby Cops Enabled
+
+We have enabled many new built-in RuboCop cops that we believe are also useful in correcting Chef Infra cookbooks:
+
+#### Lint/BinaryOperatorWithIdenticalOperands
+
+The `Lint/BinaryOperatorWithIdenticalOperands` cop detects conditionals with duplicate checks. For example `b == 0 && b == 0` which can be simplified to just `b == 0`
+
+#### Lint/DuplicateElsifCondition
+
+The `Lint/BinaryOperatorWithIdenticalOperands` cop detects conditions with duplicate `elsif` checks that can be removed.
+
+#### Lint/DuplicateRescueException
+
+The `Lint/BinaryOperatorWithIdenticalOperands` cop detects duplicate `rescue` statements in code that can be removed.
+
+#### Lint/OutOfRangeRegexpRef
+
+The `Lint/OutOfRangeRegexpRef` detects the use of regex captures that are outside of the range of possible within a regex statement. For example `/(foo)bar/ =~ 'foobar'` has a single capture so `$2` is not a valid capture and will always return `nil`.
+
+#### Style/ArrayCoercion
+
+The `Style/ArrayCoercion` cop ensures a consistent method of coercing data types into Arrays. For example this will autocorrect `foo = "some_string"; [foo]` into `foo = "some_string"; Array(foo)` and `[*paths]` into `Array(paths)` to make it more clear what is being done.
```

Comment:
```suggestion
The `Style/ArrayCoercion` cop ensures a consistent method of coercing data types into Arrays. For example, this will autocorrect `foo = "some_string"; [foo]` into `foo = "some_string"; Array(foo)` and `[*paths]` into `Array(paths)` to make it more clear what is being done.
```